test/ketu/async/integration_test.clj

```diff
@@ -208,3 +208,38 @@
                {:assigned [1] :consumer-assignment [1]}}
              (hash-set (u/try-take! assignment-chan) (u/try-take! assignment-chan))))
       (source/stop! source))))
+
+(deftest commands-channel
+  (let [commands-chan (async/chan 10)
+        consumer-chan (async/chan 10)
+        result-chan (async/chan 10)
+        clicks-consumer-opts {:name                   "clicks-consumer"
+                              :brokers                "127.0.0.1:9999"
+                              :topic                  "clicks"
+                              :group-id               "clicks-test-consumer"
+                              :auto-offset-reset      "earliest"
+                              :ketu.source/consumer-commands-chan commands-chan}
```

Comment:
This repo doesn't use aligned `let` bindings, please update to single space between the keyword and the value.

Reply:
👍